### PR TITLE
[PIN-2392] Add snapshot tests for shared components

### PR DESCRIPTION
**Pull request by LorenzoCocciarelli**
### Description

Adds snapshot tests for components:
- Table.tsx
- Stepper.tsx
- StepActions.tsx
- Spinner.tsx
- MUISkeletons/ (cartella intera)
- InputWrapper.tsx
- InputError.tsx
- InlineClipboard.tsx
- InfoTooltip.tsx
- ActionMenu.tsx
- Accordion.tsx

All tests run correctly

### CODE REVIEW

File: src/components/shared/Accordion.tsx [hunk 1]
```diff
@@ -24,11 +24,13 @@ export function Accordion({ entries }: StyledAccordionProps) {
         <MUIAccordion key={i}>
           <AccordionSummary
             expandIcon={<ExpandMoreIcon />}
+            aria-label={`panel-content-${i}`}
```
Comment (ruggerocastagnola) [inline src/components/shared/Accordion.tsx:27]: This should not be here. We added it just for testing, should be removed

File: src/components/shared/__tests__/InfoTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { InfoTooltip } from '@/components/shared/InfoTooltip'
+
+describe("Checks that InfoTooltip snapshots don't change", () => {
+  it('renders correctly', () => {
+    const inlineClipboard = render(<InfoTooltip label={'label'} />)
```
Comment (ruggerocastagnola) [inline src/components/shared/__tests__/InfoTooltip.tsx:7]: Should be `infoTooltip`. @Carminepo2 , we should add a test when hovered (label displayed)

File: src/components/shared/__tests__/ActionMenu.test.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+const mockActions: Array<ActionItem> = [
+  {
+    action: vi.fn,
+    label: 'label1',
+  },
+]
+
+describe("Checks that ActionMenu snapshots don't change", () => {
+  it('renders correctly', () => {
+    const inlineClipboard = render(<ActionMenu actions={mockActions} />)
```
Comment (ruggerocastagnola) [inline src/components/shared/__tests__/ActionMenu.test.tsx:17]: Should be `actionMenu`. Also @Carminepo2 we should add more test cases

**Review by ruggerocastagnola — CHANGES_REQUESTED** [review]

File: src/components/shared/Accordion.tsx (hunk 1 above)
Comment (LorenzoCocciarelli) [reply src/components/shared/Accordion.tsx]: aria-label removed

File: src/components/shared/__tests__/InfoTooltip.tsx (hunk 2 above)
Comment (LorenzoCocciarelli) [reply src/components/shared/__tests__/InfoTooltip.tsx]: fixed name and  added hovered state snapshot

File: src/components/shared/__tests__/ActionMenu.test.tsx (hunk 3 above)
Comment (LorenzoCocciarelli) [reply src/components/shared/__tests__/ActionMenu.test.tsx]: fixed name and added other test cases

**Review by ruggerocastagnola — APPROVED** [review]

**Pull request merged**